### DISCUSSION TITLE
Copy VS Code terminal on ipad

### DISCUSSION THREAD

**Discussion by YaashikaB** [coder/code-server#6794] [Q&A]
Hi! I tried copying using the long press function as usual, but that seems not to work. Any workarounds?

I seem to be having difficulty copying more than a line of code as well.

**Comment by code-asher** [comment]
Are you able to reproduce in vscode.dev or Codespaces?  If so, I would recommend reporting upstream at https://github.com/microsoft/vscode, they should have a better idea of what is going on.

**Reply by YaashikaB** [reply]
Hi! I’m sorry but what does reproduce mean? I’ve just started coding.

**Reply by code-asher** [reply]
No worries, reproduce in this context means to make the bug/problem happen again.

code-server is a project that patches and wraps around VS Code Web, so most bugs are in VS Code and not code-server.  VS Code Web is used in GitHub Codespaces, so we can use that to test if the bug is theirs or ours. 

If you go to https://github.com/codespaces you should be able to create a blank workspace (click "Use this template" under "Blank").

Then, see if you get the same bug in the terminal.  If you do, then you can report the issue at https://github.com/microsoft/vscode.

I would try it but I do not have an iPad on me at the moment.

**Reply by YaashikaB** [reply]
Hi, thanks for guiding me! And yes, it seems that the bug persists. I’ll report it forward.

Thanks a lot for your help again!

**Reply by YaashikaB** [reply]
Another thing I’d like to add, I’m using the VS code provided by Harvard’s CS50x course. Would I have to report elsewhere or to Microsoft VS Code itself?

**Reply by code-asher** [reply]
If you can reproduce it in Codespaces, report it to VS Code itself!
